modules/diversion_qr_to_waypoint_list.py

```diff
@@ -0,0 +1,50 @@
+"""
+Function to parse diversion QR string into a list of vertices and a rejoin waypoint.
+"""
+import re
+
+
+def diversion_qr_to_waypoint_list(qr_text: str) -> "tuple[bool, tuple[list[str], str] | None]":
+    """
+    Function for parsing QR text into list of diversion waypoints and a rejoin waypoint.
+    Input string format: "Avoid the area bounded by: Name1; Name2; Name3; Name4. Rejoin the route at Waypoint5"
+
+    Parameters
+    -----------
+    qr_text: str
+    Diversion QR string which needs to be parsed.
+     

```

Comment:
I meant, keep the indentation for line 15, but remove the spaces on line 16 (leave an empty line).